### ISSUE TITLE
[AnVIL Next] Implement findable-ui header 

### ISSUE THREAD

**Issue by MillenniumFalconMechanic** [anvilproject/anvil-portal#2677]
#### Needs
The consortia list and detail pages are to be built out on a new Next.JS app rather than the existing Gatsby platform. The existing functionality and content on the AnVIL Portal Gatsby site will eventually be ported to this new Next.JS but we are starting with the consortia functionality.

#### Depends On
*  [Next.JS implementation](https://github.com/clevercanary/anvil-portal/issues/4) must be completed before this task can be started.

#### Approach
* Add findable-ui package
* Add/configure header component 
* Header should match current AnVIL Portal header, with the addition of a "Consortia" link. See [mocks](https://www.figma.com/file/JOUDXkKfgivRNFdBsWDPEH/AnVIL-Data-Explorer?type=design&node-id=2529-27637&t=3W26OwqbFE4xIjz0-0).
* Consortia links to consortia overview page ([mocks](https://www.figma.com/file/JOUDXkKfgivRNFdBsWDPEH/AnVIL-Data-Explorer?type=design&node-id=2670-27422&t=jiibOC26nfX4FX1v-0)).

<img width="737" alt="Screen Shot 2023-05-11 at 9 26 35 AM" src="https://github.com/anvilproject/anvil-portal/assets/2446424/5db14cae-875e-418b-a6cf-759f420ab708">

#### Out of Scope
* Consortia menu (see below) is **out of scope** for this ticket.
<img width="231" alt="Screen Shot 2023-05-11 at 9 23 47 AM" src="https://github.com/anvilproject/anvil-portal/assets/2446424/489ec1cc-9c5e-4022-91b7-49eba3c38101">

**Comment by MillenniumFalconMechanic** [comment]
Completed in https://github.com/anvilproject/anvil-portal/pull/2706.
